src/young_tableau/young_tableau.hpp

```diff
@@ -0,0 +1,793 @@
+struct Projector<
+        YoungTableauSeq<std::index_sequence<ElemOfHeadRow...>, TailRow...>,
+        Dimension,
+        AntiSym>
+{
+    template <class... Id>
+    static sil::tensor::Tensor<
+            double,
+            ddc::DiscreteDomain<Id...>,
+            std::experimental::layout_right,
+            Kokkos::DefaultHostExecutionSpace::memory_space>
+    run(sil::tensor::Tensor<
+            double,
+            ddc::DiscreteDomain<Id...>,
+            std::experimental::layout_right,
+            Kokkos::DefaultHostExecutionSpace::memory_space> proj)
+    {
+        if constexpr (sizeof...(ElemOfHeadRow) >= 2) {
+            // Allocate & build a symmetric projector for the row
+            sil::tensor::TensorAccessor<symmetrizer_index_t<Id, Id...>...> sym_accessor;
+            ddc::DiscreteDomain<symmetrizer_index_t<Id, Id...>...> sym_dom
+                    = sym_accessor.mem_domain();
+
+            ddc::Chunk sym_alloc(sym_dom, ddc::HostAllocator<double>());
+            sil::tensor::Tensor<
+                    double,
+                    ddc::DiscreteDomain<symmetrizer_index_t<Id, Id...>...>,
+                    std::experimental::layout_right,
+                    Kokkos::DefaultHostExecutionSpace::memory_space>
+                    sym(sym_alloc);
+            ddc::parallel_fill(sym, 0);
+            std::array<std::size_t, sizeof...(Id) / 2> idx_to_permute;
+            for (std::size_t i = 0; i < sizeof...(Id) / 2; ++i) {
+                idx_to_permute[i] = i;
+            }
+            std::array<std::size_t, sizeof...(ElemOfHeadRow)> row_values {ElemOfHeadRow...};
+            auto idx_permutations = detail::permutations_subset(idx_to_permute, row_values);
+            for (int i = 0; i < idx_permutations.size(); ++i) {
+                fill_symmetrizer<
+                        Dimension,
+                        AntiSym,
+                        symmetrizer_index_t<Id, Id...>...>(sym, idx_permutations[i]);
+            }
+
+            // Extract the symmetric part (for the row) of the projector (requires an intermediate prod tensor)
+            ddc::Chunk prod_alloc(tensor_prod_domain(sym, proj), ddc::HostAllocator<double>());
+            sil::tensor::Tensor<
+                    double,
+                    sil::tensor::tensor_prod_domain_t<
+                            ddc::DiscreteDomain<symmetrizer_index_t<Id, Id...>...>,
+                            ddc::DiscreteDomain<Id...>>,
+                    std::experimental::layout_right,
+                    Kokkos::DefaultHostExecutionSpace::memory_space>
+                    prod(prod_alloc);
+            sil::tensor::tensor_prod(prod, sym, proj);
+            Kokkos::deep_copy(
+                    proj.allocation_kokkos_view(),
+                    prod.allocation_kokkos_view()); // We use Kokkos::deep_copy in place of ddc::parallel_deepcopy to avoid type verification of the type dimensions
```

Comment:
```suggestion
                    prod.allocation_kokkos_view()); // We rely on Kokkos::deep_copy in place of ddc::parallel_deepcopy to avoid type verification of the type dimensions
```